content/docs/get-started/index.md

```diff
@@ -1,20 +1,24 @@
 ---
-description: 'Learn how you can use MLEM to easily manage and deploy models'
+description:
+  'Learn how to use MLEM to manage and deploy machine learning models easily.'
 ---
 
-# Get Started
+# Get Started with MLEM
 
-For this guide, we will need a Python environment with the following python
-packages installed: `pandas`, `scikit-learn`, `mlem[fastapi,heroku]`.
-Additionally, we'll require `docker` to be installed on your local machine to
-showcase building and deploying a containerized model server.
+<!--
+## Get Started with MLEM
+-->
```

Comment:
```suggestion
```

Reply:
This is a trick so that the header is shown in the ToC (see pic below). I've been using these in DVC docs as a workaround to https://github.com/iterative/gatsby-theme-iterative/issues/37 for now -- we can batch remove them when/if that gets addressed.

<img width="822" alt="image" src="https://user-images.githubusercontent.com/1477535/209417153-a5773641-e2c2-4dda-b5d1-4685f6137b36.png">